Use resin-u-boot bbclass

floion opened the pull request.

Commit:
systemd_%.bbappend: Do not add the watchdog conf
file to the rootfilesystem

Commit 635a0c2975e17c3ef8b2b6faceb4f961d63f3290
mistakenly removed this file.
Re-adding it now

Signed-off-by: Florin Sarbu <florin@resin.io>

Commit:
u-boot-ts_git.bb: Change the SD boot
kernel cmdline to include console and
enable_wait_mode params

This way we are consistent with the eMMC boot

Signed-off-by: Florin Sarbu <florin@resin.io>

Commit:
u-boot-ts_git.bb: Integrate with resin uboot

Signed-off-by: Florin Sarbu <florin@resin.io>

Commit:
resin-image-flasher.bbappend: Remove
boot.ub binary config file

We are using resin-u-boot.bbclass now instead

Signed-off-by: Florin Sarbu <florin@resin.io>

Commit:
u-boot-script-ts.bb: Remove recipe

We are using resin-u-boot.bbclass now so there is no need
to generate a separate binary config file for when booting
off the SD card.

Signed-off-by: Florin Sarbu <florin@resin.io>

Commit:
CHANGELOG.md: Add entry for resin-uboot
support for the ts4900 board

Signed-off-by: Florin Sarbu <florin@resin.io>

agherzan commented on layers/meta-resin-ts/recipes-bsp/u-boot/u-boot-ts/0004-Replace-default-sd-bootargs.patch:
```diff
@@ -0,0 +1,19 @@
+ 		"ice40 ${loadaddr} ${filesize}; " \
+ 		"load mmc 0:1 ${loadaddr} ${uimage}; " \
+-		"setenv bootargs root=/dev/mmcblk1p1 rootwait rw ${cmdline_append}; " \
++		"setenv bootargs root=/dev/mmcblk1p1 rootwait rw console=ttymxc0,115200 enable_wait_mode=off; " \
```
we would probably want this ro though.

floion replied:
The rootfs is remounted ro later on in the boot process

telphan replied:
Still, why not mount it ro here?

floion replied:
Because that change has no effect.
If there is consensus to change it though, I will.
@agherzan what is the decision here?

agherzan replied:
We don't have that in the specification. It was just an idea. Don't think it's that important at this point.

agherzan commented on layers/meta-resin-ts/recipes-core/systemd/systemd_%.bbappend:
```diff
@@ -0,0 +1,3 @@
+do_install_append_ts4900() {
```
Can you please reference the commit where this mistake was included?

floion replied:
Done

agherzan commented on layers/meta-resin-ts/recipes-bsp/u-boot/u-boot-ts/ts4900-integrate-with-resin-configuration.patch:
```diff
@@ -0,0 +1,57 @@
++				"then run emmcboot; " \
++			"fi;" \
++		"fi;" \
++		"else run emmcboot; " \
```
This looks badly indented.

floion replied:
I indented like this for clarity.
Please paste the way you want it indented

agherzan replied:
What if is this else supposed to be part of?

floion replied:
if test ${jpsdboot} = 'off'

Commit:
meta-resin: Update this
sumbodule to master HEAD

We need this so that we can use the resin-u-boot.bbclass

Signed-off-by: Florin Sarbu <florin@resin.io>
